VideoWeb/VideoWeb/ClientApp/src/app/services/error.service.ts

```diff
@@ -44,8 +53,16 @@ export class ErrorService {
         }
     }
 
+    private async checkIntenetConnection(): Promise<void> {
+        try {
+            await this.profileService.getProfileByUsername(this.adalService.userInfo.userName);
```

Comment:
you can remove the need for AdalService by calling `getUserProfile` in the profile service, this does the same thing without relying on the username being manually provided.

Reply:
removed the adal service and changed call to getUserProfile